SampleApps/QuickstartExample/Quickstart.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -0,0 +1,25 @@
+{
+  "object": {
+    "pins": [
+      {
+        "package": "Facebook",
+        "repositoryURL": "https://github.com/facebook/facebook-ios-sdk.git",
+        "state": {
+          "branch": null,
+          "revision": "82dec0403c32d7b1d93529dcabb9fcb18f0825cc",
+          "version": "9.1.0"
+        }
+      },
+      {
+        "package": "ForgeRock-iOS-SDK",
```

Comment:
I prefer it been in SPM or CocoaPods as when following the tutorial we lead them to do that.
In theory the SPM if the SDK is added an SPM package it can pull the latest every time

Reply:
👍🏻